website/docs/r/git_repository.html.markdown

```diff
@@ -68,9 +84,11 @@ The following arguments are supported:
 
 `initialization` - (Required) block supports the following:
 
-- `init_type` - (Required) The type of repository to create. Valid values: `Uninitialized`, `Clean` or `Import`. Defaults to `Uninitialized`.
+- `init_type` - (Required) The type of repository to create. Valid values: `Uninitialized`, `Clean` or `Import`.
```

Comment:
this is a required field and doesn't actually default to Unitialized, so I removed it.